### ISSUE TITLE
Using --params-file command line option should raise an error but only a warning is raised

### ISSUE THREAD

**Issue by vmkalbskopf** [nextflow-io/nf-schema#171]
If a user mistakenly uses two dashes (as opposed to one dash) when specifying the params file, the plugin should raise an error, but only a warning is raised in a long list of other warnings.

Command used and terminal output
```
nextflow run nf-core/rnaseq -r 3.20.0 -profile singularity --params-file params.yaml

WARN: The following invalid input values have been detected:

* --custom_config_base: https://raw.githubusercontent.com/nf-core/configs/master
* --kallisto_quant_fraglen_sd: 200
* --featurecounts_feature_type: exon
* --ribo_database_manifest: /home/bioinformatics/.nextflow/assets/nf-core/rnaseq/workflows/rnaseq/assets/rrna-db-defaults.txt
* --extra_fqlint_args: --disable-validator P001
* --aligner: star_salmon
* --custom_config_version: master
* --featurecounts_group_type: gene_biotype
* --hisat2_build_memory: 200.GB
* --skip_preseq: true
* --umi_dedup_tool: umitools
* --gtf_extra_attributes: gene_name
* --max_multiqc_email_size: 25.MB
* --bracken_precision: S
* --min_trimmed_reads: 10000
* --pseudo_aligner_kmer_size: 31
* --deseq2_vst: true
* --umitools_extract_method: string
* --validate_params: true
* --min_mapped_reads: 5
* --rseqc_modules: bam_stat,inner_distance,infer_experiment,junction_annotation,junction_saturation,read_distribution,read_duplication
* --gtf_group_features: gene_id
* --kallisto_quant_fraglen: 200
* --paramsFile: params.yaml
* --trimmer: trimgalore
* --skip_bbsplit: true
* --publish_dir_mode: copy
* --pipelines_testdata_base_path: https://raw.githubusercontent.com/nf-core/test-datasets/7f1614baeb0ddf66e60be78c3d9fa55440465ac8/
* --stranded_threshold: 0.8
* --igenomes_base: s3://ngi-igenomes/igenomes/
* --trace_report_suffix: 2025-08-20_10-51-42
* --unstranded_threshold: 0.1
* --umitools_grouping_method: directional


ERROR ~ Validation of pipeline parameters failed!

 -- Check '.nextflow.log' file for details
The following invalid input values have been detected:

* Missing required parameter(s): input, outdir
```

System information
N E X T F L O W ~ version 25.04.6
Desktop
local executor
Docker and singularity result in the same behaviour
Ubuntu 24.04.3 LTS
rnaseq v3.20.0

**Comment by awgymer** [comment]
Pretty obviously `paramsFile` does not match our expected `params-file` in the default nextflow options. 

I removed a camel case check but that was for checking the *inverse* issue of `camelCase` becoming `kebab-case` [#2061](https://github.com/nextflow-io/nextflow/issues/2061) which didn't actually seem to be an issue. 

If the default nextflow options are being munged from `kebab-case` to `camelCase` then we will need to look at adding some other test, but I'd like some confirmation that this is expected behaviour. 

@ewels any idea?

**Comment by awgymer** [comment]
Per comments from Ben on that original issue, `kebab-case` is no longer transformed to `camelCase`, but the reverse is true. 

We should implement a check for this with the default nextflow params. 

Although it is theoretically possible for someone to also use kebab-case in `params` (and thus we would also need to account for this more generally) it's actually non-trivial to set a param in the config as `kebab-case`, `snake_case` is much more common and does not have this issue.
